addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -33,6 +34,15 @@ class PaymentAcquirer(models.Model):
         help="If a webhook is enabled on your Stripe account, this signing secret must be set to "
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
+    registered_domain_name = fields.Char(
+        string="Apple Pay: Registered domain name",
+        help="The domain name registered by Stripe where Apple Pay is allowed to be used.",
+    )
+    is_domain_name_registered = fields.Boolean(
+        string="Is the domain name registered",
+        help="Whether the domain name is registered in Stripe's Backend or not.",
+        compute='_compute_is_domain_name_registered',
+    )
```

Comment:
To discuss with @pierre-pde 

And move Apple Pay register domain to the configuration tab

Reply:
Hm, thinking of it this morning, 2 buttons'd be overkill too, as is the `registered_domain_name` field, this is literally the `base_url`, why should you store it again?

One action button can stay that'll either send the base url/check the url received from Stripe is the base_url (or you check if it's okay from Stripe, if not, you send the base url). But these fields are useless IMHO.

As for the visual aspect on the configuration/credentials page, I think that just having the button with "Verify Apple Pay compatibility" or something like this would be enough, with a rainbowman effect when everything is ok, and a userError if not.